### PR TITLE
store/mockstore/unistore/tikv: migrate test-infra to testify

### LINKED ISSUE #27652: migrate test-infra to testify for `store/mockstore`

### PR DESCRIPTION
**Pull request by unconsolable**
### What problem does this PR solve?

Issue Number: close #27652

### What is changed and how it works?

What's Changed:

- Migrate predicates to testify

### Release note 

```release-note
None
```

### CODE REVIEW

**Comment by unconsolable** [comment]
/cc @tisonkun 
PTAL

**Review by tisonkun — APPROVED** [review]

File: store/mockstore/unistore/tikv/mvcc_test.go [hunk 1]
```diff
@@ -1217,11 +1198,11 @@ func (s *testMvccSuite) TestMinCommitTs(c *C) {
...
 }
 
-func (s *testMvccSuite) TestGC(c *C) {
-	c.Skip("GC work is hand over to badger.")
-	store, err := NewTestStore("TestGC", "TestGC", c)
-	c.Assert(err, IsNil)
-	defer CleanTestStore(store)
+func TestGC(t *testing.T) {
+	t.Parallel()
+	t.Skip("GC work is hand over to badger.")
```
Comment (disksing) [inline store/mockstore/unistore/tikv/mvcc_test.go:1203]: can we remove this test case?

**Review by disksing — APPROVED** [review]

**Review by tisonkun — APPROVED** [review]

**Pull request merged**